api/v1beta2/awsmachine_types.go

```diff
@@ -193,6 +193,11 @@ type AWSMachineSpec struct {
 	// PrivateDNSName is the options for the instance hostname.
 	// +optional
 	PrivateDNSName *PrivateDNSName `json:"privateDnsName,omitempty"`
+
+	// CapacityReservationID specifies the instance that should be launched in the
+	// reserved compute capacity.
```

Comment:
This does not specify an instance.

```suggestion
	// CapacityReservationID specifies the target Capacity Reservation into which the instance should be launched.
```